pom.xml

```diff
@@ -56,94 +65,124 @@
         </contributor>
     </contributors>
 
-    <licenses>
-        <license>
-            <name>EPL 2.0</name>
-            <url>http://www.eclipse.org/legal/epl-2.0</url>
-            <distribution>repo</distribution>
-        </license>
-        <license>
-            <name>GPL2 w/ CPE</name>
-            <url>https://www.gnu.org/software/classpath/license.html</url>
-            <distribution>repo</distribution>
-        </license>
-    </licenses>
-
+    <scm>
+        <connection>scm:git:ssh://git@github.com/eclipse-ee4j/glassfish-ha-api.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com/eclipse-ee4j/glassfish-ha-api.git</developerConnection>
+        <url>https://github.com/eclipse-ee4j/glassfish-ha-api</url>
+        <tag>HEAD</tag>
+    </scm>
     <issueManagement>
         <system>github</system>
         <url>https://github.com/eclipse-ee4j/glassfish/issues</url>
     </issueManagement>
 
     <properties>
-        <hk2.version>3.0.2</hk2.version>
-        <legal.doc.source>${maven.multiModuleProjectDirectory}</legal.doc.source>
-        <base.java.level>8</base.java.level>
-        <upper.java.level>9</upper.java.level>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <maven.compiler.release>11</maven.compiler.release>
+        <hk2.version>3.0.5</hk2.version>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.glassfish.hk2</groupId>
+            <artifactId>hk2</artifactId>
+            <version>${hk2.version}</version>
+            <optional>true</optional>
+        </dependency>
+        
+        <!-- Tests -->
+        
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.13.2</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <pluginRepositories>
+        <pluginRepository>
+            <id>dash-licenses-snapshots</id>
+            <url>https://repo.eclipse.org/content/groups/releases/</url>
+        </pluginRepository>
+    </pluginRepositories>
+
     <build>
+        <resources>
+            <resource>
+                <directory>src/main/java</directory>
+                <includes>
+                    <include>**/*.properties</include>
+                    <include>**/*.xml</include>
+                </includes>
+            </resource>
+            <resource>
+                <directory>${project.basedir}/..</directory>
```

Comment:
Hmm, that's correct path?

Reply:
Antipattern - using a directory of different module 😉

Reply:
Moreover, dir outside project root

Reply:
It was the wrong one instead; copy/paste error. I tried to make the pom consistent with other poms in EE4J, but forgot to adjust that part.